vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -776,34 +772,18 @@
       <SubType>Designer</SubType>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="boost" Version="1.76.0.0" />
+    <PackageReference Include="Microsoft.Build.Tasks.Git" Version="1.0.0" />
+    <PackageReference Include="Microsoft.SourceLink.Common" Version="1.0.0" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" />
+    <PackageReference Include="Microsoft.UI.Xaml" Version="2.6.0" />
+    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.210312.4" />
+    <PackageReference Include="$(WinUIPackageName)" Version="$(WinUIPackageVersion)" />
+    <PackageReference Include="ReactNative.Hermes.Windows" Version="0.8.0-staging3" />
```

Comment:
These should use the existing version prop instead of hardcoding.

Reply:
Does the comment also apply to CppWinRT?

Reply:
I'm not sure if we have an existing shared location for its version. If not, feel free to leave as is for now.

Reply:
I was looking at parametrizing cppwinrt these days, if you can do it in this PR that would be great

One thing that I am worried about is that IDK when PackageReference is resolved w.r.t. Imports. We need to be able to have Imports run first so that versions defined in ExperimentalFeatures.props can inform which version of a nuget package we then PackageReference. Can you do a quick test in isolation to validate this works as expected? Without this functionality, I don't think we can switch to PackageReference :-/ so it seems important to validate upfront